Cargo.toml

```diff
@@ -220,7 +220,7 @@ metrics.workspace = true
 metrics-tracing-context.workspace = true
 
 # AWS - Official SDK
-aws-sdk-s3 = { version = "1.4.0", default-features = false, features = ["behavior-version-latest", "rt-tokio"], optional = true }
+aws-sdk-s3 = { version = "1.15.0", default-features = false, features = ["behavior-version-latest", "rt-tokio"], optional = true }
```

Comment:
Note that this was necessary to get past a DispatchFailure with a source ConnectorError with source InterceptorError with name UserAgentInterceptor and source MissingApiMetadata, which seems related to https://github.com/awslabs/aws-sdk-rust/issues/1093 - and, indeed, bumping the aws-sdk-s3 version resolved the issue.